R/estimate_cuminc.R

```diff
@@ -20,29 +20,64 @@
 #' @export
 #'
 #' @examples
-#' estimate_cuminc(
-#'   tidycmprsk::trial,
-#'   strata = "trt",
-#'   CNSR = "death_cr",
-#'   AVAL = "ttdeath"
-#' ) %>%
-#'   visr() %>%
-#'   add_CI() %>%
-#'   add_risktable(statlist = c("n.risk", "cum.event"))
-
-estimate_cuminc <- function(data
+#' cuminc <-
+#'   visR::estimate_cuminc(
+#'     data = tidycmprsk::trial,
+#'     strata = "trt",
+#'     CNSR = "death_cr",
+#'     AVAL = "ttdeath"
+#'   )
+#' cuminc
+#'
+#' cuminc %>%
+#'   visR::visr() %>%
+#'   visR::add_CI() %>%
+#'   visR::add_risktable(statlist = c("n.risk", "cum.event"))
+
+estimate_cuminc <- function( data = NULL
                             ,strata = NULL
                             ,CNSR = "CNSR"
                             ,AVAL = "AVAL"
                             ,conf.int = 0.95
```

Comment:
Based on your feedback, I assume there is no default CI estimation in tidycmprsk? If you this is the case, then you need to a test for CI argument: (numeric, not numeric, when you change, the estimation changes, checks for 0<x<1

Reply:
> Based on your feedback, I assume there is no default CI estimation in tidycmprsk? 

I don't understand this comment. The CIs are estimated up front, which is why the first function needs a confidence interval argument.